packages/rsk3-utils/src/index.js

```diff
@@ -1046,6 +1051,123 @@ const soliditySha3 = function() {
     return keccak256(`0x${hexArguments.join('')}`);
 };
 
+/**
+ * Gets the RSK address from BTC private key
+ *
+ * @method getRskAddress
+ *
+ * @param {String} btcPrivateKey BTC Private key.
+
+ * @return {String} RSK address.
+ */
+const getRskAddress = function(btcPrivateKey) {
+    if (!isValidBtcPrivateKey(btcPrivateKey)) return null;
```

Comment:
Regarding @alepc253 's comment, rather than return `null`, a viable alternative would be to throw an error. For example:

```suggestion
    if (!isValidBtcPrivateKey(btcPrivateKey)) {
      throw new Error('Invalid BTC private key');
    }
```

(If you change this here, change it in the other functions too.)

The impact of this would be slightly more complex code within the calling function - it would need to use a try...catch block instead of an if block. That being said, both approaches are valid, and I would decide - for the sake of consistency - based on the convention used within the rest of rsk3.js. Let's wait for @chrisli30 to weigh in.